model/modelindexer/bulk_indexer.go

```diff
@@ -140,22 +146,24 @@ func (b *bulkIndexer) Flush(ctx context.Context) (elasticsearch.BulkIndexerRespo
 	if b.itemsAdded == 0 {
 		return elasticsearch.BulkIndexerResponse{}, nil
 	}
+	req := esapi.BulkRequest{Body: &b.buf, Header: esHeader}
```

Comment:
Is there some benefit to having this up here? No big deal, but I'd prefer to keep the request construction as it was.

Reply:
I suppose not, the compiler optimizations should take care of grouping things in the way that's best for it. I'll revert it.